### PR TITLE
zh: Update mydumper backup docs

### PR DESCRIPTION
**Pull request by onlymellb**
### What is changed, added or deleted? (Required)
Update CRD backup docs with mydumper.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?
pingcap/tidb-operator#2435
pingcap/tidb-operator#2407

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Comment by ran-huang** [comment]
Please fix the CI failure and then involve a tech reviewer.

File: zh/backup-to-gcs.md [hunk 1]
```diff
@@ -128,6 +139,21 @@ GCS 支持以下几种 bucket ACL 策略：
 * `.spec.from.port`：待备份 TiDB 集群的访问端口。
 * `.spec.from.user`：待备份 TiDB 集群的访问用户。
 * `.spec.from.tidbSecretName`：待备份 TiDB 集群所需凭证的 secret。
+* `.spec.gcs.bucket`：存储数据的 bucket 名字。
+* `.spec.gcs.prefix`：这个字段可以省略，如果设置了这个字段，则会使用这个字段来拼接在远端存储的存储路径`s3://${.spec.s3.bucket}/${.spec.s3.prefix}/backupName`。
```
Comment (ran-huang) [inline zh/backup-to-gcs.md:143]:
```suggestion
* `.spec.gcs.prefix`：这个字段可以省略，如果设置了这个字段，则会使用这个字段来拼接在远端存储的存储路径 `s3://${.spec.s3.bucket}/${.spec.s3.prefix}/backupName`。
```

File: zh/backup-to-gcs.md [hunk 2]
```diff
@@ -128,6 +139,21 @@ GCS 支持以下几种 bucket ACL 策略：
...
+* `.spec.mydumper`： mydumper 相关的配置，主要有两个字段一个是 [options](https://pingcap.com/docs-cn/stable/reference/tools/mydumper/), 里面可以指定 mydumper 需要的一些参数。一个是 tableRegex 字段可以指定让 mydumper 备份符合这个正则表达式的表。默认情况下 mydumper 这个字段可以不用配置。当不指定 mydumper 的配置时，`options` 和 `tableRegex` 字段的默认值如下：
```
Comment (ran-huang) [inline zh/backup-to-gcs.md:144]:
```suggestion
* `.spec.mydumper`：Mydumper 相关的配置，主要有两个字段：一个是 [`options`](https://pingcap.com/docs-cn/stable/reference/tools/mydumper/) 字段，里面可以指定 mydumper 需要的一些参数；一个是 `tableRegex` 字段，可以指定让 Mydumper 备份符合这个正则表达式的表。默认情况下 Mydumper 这个字段可以不用配置。当不指定 Mydumper 的配置时，`options` 和 `tableRegex` 字段的默认值如下：
```

File: zh/backup-to-s3.md [hunk 3]
```diff
@@ -218,10 +258,24 @@ Amazon S3 支持以下几种 `storageClass` 类型：
 * `.spec.from.port`：待备份 TiDB 集群的访问端口。
 * `.spec.from.user`：待备份 TiDB 集群的访问用户。
 * `.spec.from.secretName`：存储 `.spec.from.user` 用户的密码的 secret。
-* `.spec.s3.region`: 使用 Amazon S3 存储备份，需要配置 Amazon S3 所在的 region。
-* `.spec.s3.bucket`: 兼容 S3 存储的 bucket 名字。
-* `.spec.storageClassName`: 备份时所需的 persistent volume (PV) 类型。
-* `.spec.storageSize`: 备份时指定所需的 PV 大小。该值须大于 TiDB 集群备份的数据大小。
+* `.spec.s3.region`：使用 Amazon S3 存储备份，需要配置 Amazon S3 所在的 region。
+* `.spec.s3.bucket`：兼容 S3 存储的 bucket 名字。
+* `.spec.s3.prefix`：这个字段可以省略，如果设置了这个字段，则会使用这个字段来拼接在远端存储的存储路径`s3://${.spec.s3.bucket}/${.spec.s3.prefix}/backupName`。
```
Comment (ran-huang) [inline zh/backup-to-s3.md:263]:
```suggestion
* `.spec.s3.prefix`：这个字段可以省略，如果设置了这个字段，则会使用这个字段来拼接在远端存储的存储路径 `s3://${.spec.s3.bucket}/${.spec.s3.prefix}/backupName`。
```

File: zh/backup-to-s3.md [hunk 4]
```diff
@@ -218,10 +258,24 @@ Amazon S3 支持以下几种 `storageClass` 类型：
...
+* `.spec.mydumper`： mydumper 相关的配置，主要有两个字段一个是 [options](https://pingcap.com/docs-cn/stable/reference/tools/mydumper/), 里面可以指定 mydumper 需要的一些参数。一个是 tableRegex 字段可以指定让 mydumper 备份符合这个正则表达式的表。默认情况下 mydumper 这个字段可以不用配置。当不指定 mydumper 的配置时，`options` 和 `tableRegex` 字段的默认值如下：
```
Comment (ran-huang) [inline zh/backup-to-s3.md:264]:
```suggestion
* `.spec.mydumper`：Mydumper 相关的配置，主要有两个字段：一个是 [`options`](https://pingcap.com/docs-cn/stable/reference/tools/mydumper/) 字段，里面可以指定 Mydumper 需要的一些参数；一个是 `tableRegex` 字段，可以指定让 Mydumper 备份符合这个正则表达式的表。默认情况下 Mydumper 这个字段可以不用配置。当不指定 Mydumper 的配置时，`options` 和 `tableRegex` 字段的默认值如下：
```

File: zh/backup-to-gcs.md [hunk 5]
```diff
@@ -128,6 +139,21 @@ GCS 支持以下几种 bucket ACL 策略：
 * `.spec.from.port`：待备份 TiDB 集群的访问端口。
 * `.spec.from.user`：待备份 TiDB 集群的访问用户。
 * `.spec.from.tidbSecretName`：待备份 TiDB 集群所需凭证的 secret。
+* `.spec.gcs.bucket`：存储数据的 bucket 名字。
+* `.spec.gcs.prefix`：这个字段可以省略，如果设置了这个字段，则会使用这个字段来拼接在远端存储的存储路径 `s3://${.spec.s3.bucket}/${.spec.s3.prefix}/backupName`。
```
Comment (TomShawn) [inline zh/backup-to-gcs.md:143]: @onlymellb `s3://${.spec.s3.bucket}/${.spec.s3.prefix}/backupName` 里并没有出现 `.spec.gcs.prefix` 这个字段哦，是不是写错了呢？

Comment (ran-huang) [reply zh/backup-to-gcs.md:143]: Fixed in https://github.com/pingcap/docs-tidb-operator/pull/327

**Pull request merged**